config.json

```diff
@@ -6,5 +6,9 @@
   "basename": "/metagenomics",
   "enaURL": "https://www.ebi.ac.uk/ena/browser/view/",
   "hmmer": "https://wwwdev.ebi.ac.uk/metagenomics/sequence-search/search/phmmer",
-  "googleMapsKey": "XX"
+  "googleMapsKey": "XX",
+  "sentryDsn": "https://7cb28af2cf254e70ae5eb72733f745e3@o1128655.ingest.sentry.io/6171583",
```

Comment:
I don't know if you have notice that you can have a private config file that it's not pushed to git `config.private.json`
As I'm not familiar with sentry, I'm not sure if this URL should be on the private or if is OK just to be here

Reply:
Thanks. Yeh, I noticed that. But no, the Sentry key is not secret (it is present in the client code so served to any user of the website). We will potentially have different keys for each environment, but I assume we will do that by replacing this config.json during deployments, so this is just the default.

Reply:
Cool!
Yeah, one of the steps in jenkins is creating the `config.private.json` which basically overwrites the values of `config.json`. So yeah if this is a default, then it's good to have it here